src/components/singin-store-wallet/singin-store-wallet.tsx

```diff
@@ -0,0 +1,114 @@
+import React, {useEffect} from 'react';
+
+import {View} from 'react-native';
+
+import {captureException, showModal} from '@app/helpers';
+import {useTypedNavigation, useTypedRoute, useWallets} from '@app/hooks';
+import {I18N, getText} from '@app/i18n';
+import {EthNetwork} from '@app/services';
+import {restoreFromMnemonic} from '@app/services/eth-utils';
+import {WalletType} from '@app/types';
+import {ETH_HD_SHORT_PATH, MAIN_ACCOUNT_NAME} from '@app/variables';
+
+export const SigninStoreWallet = () => {
+  const navigation = useTypedNavigation();
+  const {nextScreen, mnemonic, privateKey} =
+    useTypedRoute<'restoreStore'>().params;
+  const wallets = useWallets();
+
+  useEffect(() => {
+    showModal(getText(I18N.singinStoreWalletLoading), {
```

Comment:
The first argument is the name of the modal window that needs to be shown, it should not be a translation